content/tutorials/quickstart/py-create-accounts-send-xrp.md

```diff
@@ -215,21 +215,21 @@ import json
 Import the methods from mod1.py.
 
 ```python
-from mod1 import getAccount
-from mod1 import getAccountInfo
-from mod1 import sendXRP
+from mod1 import get_account
+from mod1 import get_accountInfo
+from mod1 import send_xrp
 ```
 
 ### getStandbyAccount
```

Comment:
Just a minor one, change the `getStandbyAccount` heading to `get_standby_account`.